src/wristpy/core/cli.py

```diff
@@ -62,6 +62,36 @@ def version_check(version: bool) -> None:
         raise typer.Exit()
 
 
+def _parse_thresholds(thresholds: list[str]) -> list[tuple[float, float, float]]:
+    """Parse the threshold strings into a list of tuples.
+
+    Args:
+        thresholds: List of threshold strings, each containing three space-separated
+            floats.
+
+    Returns:
+        List of tuple float triplets containing the parsed threshold values.
+
+    Raises:
+        typer.BadParameter: If any threshold triplet does not contain exactly three
+            floats
+        typer.BadParameter: If threshold format is invalid or values cannot be parsed.
+    """
+    parsed = []
+    for triplet_str in thresholds:
+        parts = triplet_str.strip().split()
+        if len(parts) != 3:
+            raise typer.BadParameter(
+                f"Threshold triplet must have exactly 3 floats: {triplet_str}"
+            )
+        try:
+            values = [float(part) for part in parts]
+            parsed.append((values[0], values[1], values[2]))
```

Comment:
Move this outside the try statement

```suggestion
parsed.append(tuple(values))

```